rapidsms/backends/http/tests.py

```diff
@@ -116,14 +115,14 @@ def test_valid_post_message(self):
         message = self.inbound[0]
         self.assertEqual(data['text'], message.text)
         self.assertEqual(data['identity'],
-                         message.connection.identity)
+                         message.connections[0].identity)
```

Comment:
`rapidsms` itself has already deprecated `message.connection` in favor of `message.connections[0]`.